doc/code_snippets/snippets/config/instances.enabled/tcm_get_started_tt/config.yaml

```
@@ -0,0 +1,5 @@
+config:
+  etcd:
+    endpoints:
+    - http://localhost:2379
```

nit: Looks like etcd listens on `127.0.0.1` by default, so this can be changed to:

```suggestion
    - http://127.0.0.1:2379
```